pages/dkp/konvoy/2.1/choose-infrastructure/verify-installation/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle: 
+title: Verify Konvoy installation
+menuWeight: 50
+excerpt: Check DKP components to verify the status of your cluster
+enterprise: false
+beta: false
+---
+
+In this section, we show you how to verify a Konvoy installation.
+
+## Check the cluster infrastructure and nodes
+
+DKP ships with some default diagnosis tools to check your cluster (such as the `describe` command).
+You can use those tools to validate your installation.
+
+If you have not done so already, set the environment variable for your cluster name, substituting `my-dkp-cluster` with the name of your cluster:
```

Comment:
```suggestion
If you have not done so already, set an environment variable for your cluster name, substituting `my-dkp-cluster` with the name of your cluster:
```
"the" makes me feel like it's a specific requirement.

Reply:
I might even go so far as to call out that the variable means nothing to the product, that it's only used for following along with the documentation.

```suggestion
This documentation uses environment variables to simplify copying and pasting instructions.
To use this convenience, set an environment variable for your cluster name, substituting `my-dkp-cluster` with the name of your cluster:
```